### PR TITLE
add getSenders and getReceivers

### PR DESCRIPTION
**Pull request by jensengar**
We have a library that depends on RTCPeerConnection which is calling `getRemoteStreams`. This is causing a deprecation warning in ff. Seems reasonable to proxy getSenders/Receivers.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "tape": "^4.0.0",
     "testling": "^1.7.1",
     "travis-multirunner": "^3.0.0",
-    "webrtc-adapter": "^3.4.3"
+    "webrtc-adapter": "^6.0.2"
```
Comment (xdumaine) [inline package.json]: We've moved a bunch of otalk repos to have adapter as a peer dependency, and I guess didn't get this one yet? Let's do that. @legastero @fippo objections?

Comment (xdumaine) [reply package.json]: Ignore me, we already did that here, and just moved it from dep to devDep https://github.com/otalk/RTCPeerConnection/commit/598f7dd3f8bcebab665fff990bbff01570923847#diff-b9cfc7f2cdf78a7f4b91a753d10865a2

**Comment by fippo** [comment]
I *think* getSenders and getReceivers is always shimmed by adapter?

can we split the travis update into a different PR?

**Comment by xdumaine** [comment]
@fippo it is, but since we don't bundle or enforce specific adapter version yet, this would be failsafe against an older version. We can deprecate proxying getRemoteStreams/getLocalStreams in this lib at some point as well.

**Comment by fippo** [comment]
get*Streams + co are extremly hard to get rid of unfortunately :-(

**Comment by jensengar** [comment]
@fippo you want the proxying of getSenders/Receivers and the travis fix in separate prs?

**Comment by jensengar** [comment]
If this is good, can someone merge it please?

**Comment by xdumaine** [comment]
I can merge and release when I get in in a couple hours.

**Comment by xdumaine** [comment]
Plublished as v8.1.0

**Pull request merged**